packages/asl-projects/client/components/trainingRecordsModal.js

```diff
@@ -222,6 +222,7 @@ export default function TrainingRecordModal({
           <div className="govuk-grid-row">
             {/* Left side with tabs */}
             <div className="govuk-grid-column-one-half">
+              {!showPrevTab && !showFirstTab && <h3>Previous version</h3>}
               {(showPrevTab || showFirstTab) && (
                 <nav className="govuk-tabs">
                   <ul>
```

Comment:
It should be clear that these are mutually exclusive
```suggestion
              {!showPrevTab && !showFirstTab 
              ? <h3>Previous version</h3>
              : <nav className="govuk-tabs">
                  <ul>
```